llvm/test/tools/llvm-objdump/X86/phdrs-lma3.test

```diff
@@ -0,0 +1,54 @@
+# RUN: yaml2obj %s > %t
+
+## Check when vma is the same, we could still print LMA correctly when dumping
```

Comment:
Perhaps more specific: .data and .data_copy have the same VMA but different sh_offset values. Check that we can still print LMA correctly.